airflow/providers/jira/sensors/jira.py

```diff
@@ -60,7 +60,7 @@ def poke(self, context: 'Context') -> Any:
         jira_result = getattr(resource, self.method_name)(**self.method_params)
         if self.result_processor is None:
             return jira_result
-        return self.result_processor(context, jira_result)
```

Comment:
Is there a reason we're not removing `context` from the `poke` method signature?

Reply:
`poke` on other sensors take `context`, so this probably should as well.